hack/test-templates.sh

```diff
@@ -192,10 +192,11 @@ if [ "$got" != "$expected" ]; then
 fi
 
 INFO "Testing limactl copy command"
-tmpfile="$HOME/lima-hostname"
+tmpdir="$(mktemp -d /var/tmp/lima-test-templates.XXXXXX)"
```

Comment:
Based on the discussion, we should use here:

    tmpdir="$(mktemp -d ${TMPDIR:-/tmp}/lima-test-templates.XXXXXX)"